mtags/src/main/scala-2/scala/meta/internal/pc/MetalsGlobal.scala

```diff
@@ -194,80 +195,126 @@ class MetalsGlobal(
       isVisited += key
       val result = tpe match {
         case TypeRef(pre, sym, args) =>
-          val ownerSymbol = pre.termSymbol
-          def hasConflictingMembersInScope =
-            history.lookupSymbol(sym.name).exists {
-              case _: LookupSucceeded => true
-              case _ => false
-            }
+          if (history.isSymbolInScope(sym, pre)) {
```

Comment:
[note]
- https://github.com/scalacenter/scalafix/commit/f2a8bcda692ae04b37553a7ba16b08d8e7bc7a69
- https://github.com/scalacenter/scalafix/commit/6af6cfc6dcb1c3fdd573b9b612b5f72378a7a0db

> Previously, the rewrite would insert names like `Value` in the position
of a path dependent type such as `Day.Value` even if the `Value` in
scope had a conflicting type `Bool.Value`. Now, when querying the
scope we take into account the prefix from where the symbols in scope
origin.

> The implementation in this PR is probably incorrect for some cases. I
had to use a hacky workaround for the custom `PrettyType` type. Either
way, this diff makes the test suite pass and allow us to move on.

👍